wiki/Game_mode/osu!catch/en.md

```diff
@@ -1,78 +1,84 @@
+## Song Selection
 
-## ![](/wiki/shared/mode/catch.png) Gameplay
+To access the osu!catch game mode press `Ctrl` + `3` at the same time, or use the the `Mode` button and select `osu!catch` in osu!(stable) or click the ![](/wiki/shared/mode/catch.png) icon on the top of the screen for osu!(lazer).
 
-### Song Selection
+To download more beatmaps for osu!catch, head to the [beatmap listing](https://osu.ppy.sh/beatmapsets) on the osu! website, or while in osu!(lazer) press `Ctrl` + `B` or click the `Beatmap listing` button 
 
-To access the osu!catch game mode, press `Ctrl` + `3` at the same time.
+![osu!(lazer) beatmap listing button](/wiki/shared/lazer_beatmap_listing.png)
 
-Alternatively, click on the `Mode` button and select `osu!catch`.
+## Player control
 
-### Gameplay Basics
+In osu!catch, the player is supposed to catch falling fruits by controlling a **platter** (also known as a "catcher") to the rhythm of the music. By default the keys to control the platters movements are:
 
-#### Playfield
+- the left `<-` and right `->` or `Z` and `X` keys to move the catcher left and right respectively.
+- Press and hold the `leftShift` key or the Left Mouse Button to activate a *dash* state which is recognizable by it making the platter *glow brightly* and leaving an afterimage while moving. Moving while in the dash state makes the player move **2 times faster** then normally.
 
-![osu!catch playfield difference between CS2 and CS7](/wiki/shared/Catch_Playfield_27.jpg "osu!catch playfield difference based on CS")
+The placement of the mouse cursor *does not* affect gameplay unless [Relax](/wiki/Gameplay/Game_modifier/Relax) mod is enabled, which makes the platter directly follow the cursor. 
 
-The zoom size of the playfield is based on CS (Circle Size) settings, with lower CS means much more zoomed in and smaller space, while higher CS means much more zoomed out and larger movement space.
+## Hitobjects
 
-The catcher can move from left to right, with the fruits falling from top to bottom. The catcher cannot move pass the left and right walls of the playfield.
+osu!catch has 3 main [hitobjects](/wiki/gameplay/Hit_object) the player interacts with:
 
-All fruits will stay at the plate until its combo ends, which the fruits will bounce out of the plate. Timing and where it falls on the plate does not matter; either a full score for successful catch or miss.
+- Fruits
+- Juice Streams
+- Banana Showers
 
-#### Fruits
+### Fruits
 
 ![Fruits](/wiki/shared/Catch_fruits.jpg "osu!catch fruits")
 
-To catch these fruits, make sure it falls to the plate and not over the plate. Collected fruits will always give 300 points each, increment combo by 1, and treated as 300 in result screen.
+**Fruits** are the simplest object type in osu!catch, all the user is required to do is move the platter under the fruit so it will fall on to the plate rather than over it.
```

Comment:
```suggestion
**Fruits** are the simplest object type in osu!catch. All the user is required to do is move the platter under the fruit so it will fall on to the plate rather than over it.
```